src/main/java/Duke.java

```diff
@@ -1,10 +1,229 @@
+//A0202021L
+
+import duke.*;
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
+    //Main Program
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String file1 = "data/duke.txt";
+        ArrayList<Task> list = new ArrayList<Task>();
+        int num = 0;
+
+        num = importFileContents(file1, list);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+        Scanner sc = new Scanner(System.in);
+        String command = sc.nextLine();
+
+        while (!command.equals("bye")) {
```

Comment:
Would it be better to make "bye" into a constant instead? This also applies for "list", "done", "todo" etc. in the switch case.